docs/source/package_reference/helpers.md

```diff
@@ -2,7 +2,7 @@
 rendered properly in your Markdown viewer.
 -->
 
-# Document Title
+# Helper methods
```

Comment:
Maybe we can put this section under API reference -> Utilities?

Reply:
Like [this](https://github.com/huggingface/peft/pull/1986/commits/6404cc67957bf5953de7036ce15ddb9ad6ebb10d)?

Reply:
Beautiful! 👍